src/components/backend-ai-settings-view.ts

```diff
@@ -351,6 +358,55 @@ export default class BackendAiSettingsView extends BackendAIPage {
             </div>
           </div>
         </lablup-activity-panel>
+        <backend-ai-dialog id="env-dialog" fixed backdrop persistent>
+          <span slot="title" class="horizontal layout center">${_t('settings.SchedulerDefault')}</span>
+          <span slot="action">
+            <mwc-icon-button icon="info" @click="${(e) => this._showConfigDescription(e, 'default')}" style="pointer-events:auto;"></mwc-icon-button>
+          </span>
+          <div slot="content" class="vertical layout" style="width: 100%;">
+            <mwc-select id="scheduler-switch" required label="${_t('settings.SchedulerType')}" 
```

Comment:
If the user didn't select scheduler and click update button, "Successfully Updated" snack bar message pops out.
We need to show each input field that is required and If any input is empty, then we should show an error message right below the input and disable the update request.